public/data/Lavakush Biyani.json

```
@@ -0,0 +1,40 @@
+{
+  "name": "Lavakush Biyani",
+  "type": "personal",
+  "bio": "I am Lavakush Biyani, currently pursuing my B.tech. I am a tech enthusiast & an open-source contributor. I am always open to collaborating on projects and innovative/disruptive ideas",
+  "avatar": "https://github.com/lavakush07.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/lavakush07",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/lavkushbiyani1",
+      "icon": "twitter"
+    },
+    {
+     "name":"Follow me on Linkedin",
+      "url":"https://www.linkedin.com/in/lavkush-biyani-07/",
+      "icon": "Linkedin"
+    }
+  ],
+   "milestones": [
+    {
+      "title": "Started Web devlopment",
+      "date": "April 2021",
+      "icon": "code",
+      "color": "grey",
+      "description": "Started building websites"
+    },{
+      "title":"Started competitive programming",
+      "date":"December 2020",
+      "icon":"code",
+      "color":"blue",
+      "description":"Started participating on codechef,Geeks for Geeks"
```

I think it'd look better like this 😅 What do you think? 
```suggestion
      "description":"Started participating on CodeChef, Geeks for Geeks"
```